### ISSUE TITLE
Prices shifted by one hour

### ISSUE THREAD

**Issue by shaman79** [rnovacek/homeassistant_cz_energy_spot_prices#47]
It seems that the prices are shifted by one hour. It may be a timezone problem, but I have HA currently set Central European Time/Prague+1:00 which should be OK.

Look at the example - OTE and raw data side by side. It shows that the price 1.2CZK/KWh starts at 09:00

![image](https://github.com/rnovacek/homeassistant_cz_energy_spot_prices/assets/35869063/8a62aded-5d0b-4052-b096-45b6ccfc0033)

Here is the time from HA where it starts at 08:00.

![image](https://github.com/rnovacek/homeassistant_cz_energy_spot_prices/assets/35869063/ce00d865-5915-4a9d-acc5-ca6f0ca69986)

Basically HA shows the price one hour earlier than it should.

**Comment by rnovacek** [comment]
I think HA shows correct price but you might be interpreting OTE prices incorrectly. "9. hodina" on OTE means 10:00 - 11:00. The day starts with "1. hodina" and it means 0:00 - 1:00. Does it make sense?

**Comment by rnovacek** [comment]
> > I think HA shows correct price but you might be interpreting OTE prices incorrectly. "9. hodina" on OTE means 10:00 - 11:00. The day starts with "1. hodina" and it means 0:00 - 1:00. Does it make sense?
> 
> just a little correction: 9th hour is from 8:00-9:00

Of course :facepalm: Thank you

**Comment by rnovacek** [comment]
Closing now. @shaman79 please reopen if you still face some problems.
